### PR TITLE
91 privacy policy html attach to link

### PR DESCRIPTION
**Pull request by hussojo**
added privacypolicy.html file to be served from frontend.

next up:
- proper text for file
- modifying register page link/text to be more informative

### CODE REVIEW

**Review by azarSarikhani — APPROVED** [review]
looks super good!

File: frontend/privacypolicy.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    </section>
+
+    <section>
+        <h2>Limitation of Liability</h2>
+        <p>We shall not be liable for any special, indirect, incidental, or consequential damages, including without limitation, damages resulting from loss of use, data, or profits, arising out of the use, inability to use, or the results of use of the website or mobile app, any websites linked to it, or any materials posted on it.</p>
+    </section>
+
+    <section>
+        <h2>Governing Law</h2>
+        <p>These terms and conditions shall be governed by and construed in accordance with the laws of [Your Country]. Any dispute arising out of or in connection with these terms and conditions shall be subject to the exclusive jurisdiction of the courts of [Your Country].</p>
```
Comment (dorianjagusch) [inline frontend/privacypolicy.html:45]: These [Your Country] should maybe already say Finland.

**Review by dorianjagusch — APPROVED** [review]
We talked about that it would be ok, to have the policy as a separate page as that would make our lives easy by forcing the user to click the link and open a separate page. Upon further thinking about it, I fear that that may break the SAP nature of the website. It is fine for now and the changes can go in (besides maybe changing the country). But we should probably discuss in more detail how we can handle it. Sorry for not thinking about it properly before....

**Review by dorianjagusch — APPROVED** [review]

**Comment by merituulie** [comment]
@hussojo was there still something coming in here or is this ready to merge?

**Pull request closed without merging**